app/components/NoticeFormat.tsx

```diff
@@ -82,7 +99,10 @@ export function NoticeFormat({
               aria-checked={currentValue === value}
               aria-describedby={`${value}-label`}
               outline={currentValue !== value}
-              onClick={() => setCurrentValue(value)}
+              onClick={() => {
+                setCurrentValue(value)
+                if (stateFunction) stateFunction(value)
```

Comment:
Use optional chaining:

```suggestion
                stateFunction?.(value)
```